docs/shapes.md

```diff
@@ -1,11 +1,36 @@
 # Shapes and layout
 
+## Structure of an XLA Op
+
+Consider an example Op:
+
+```
+%fusion.3 = bf16[32,32,4096]{2,1,0:T(8,128)(2,1)S(1)} fusion(bf16[32,32,8192]{2,1,0:T(8,128)(2,1)S(1)} %fusion.32), kind=kCustom, calls=%all-reduce-scatter.3
+```
+
+This consists on the following main components:
+
+* Op Name: `fusion.3`
```

Comment:
The `%` sign is actually part of the name and may or may not be there. It does not have any special meaning.

Reply:
Thanks! Included this in the description.